app/main.py

```diff
@@ -1,8 +1,56 @@
+from typing import Any
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_list: list) -> Any:
+        result = 0
+        for car in cars_list:
+            if car.clean_mark is None:
+                result += 0
```

Comment:
There is no need to add 0 to your result, this action will not make any change. Simply skip this condition and only satisfy the condition in `car.clean_mark < self.clean_power` case